docs/enterprise-guide/serialization.md

```diff
@@ -51,12 +51,15 @@ Currently, you can only load dumps into a Metabase instance that were created fr
 
 `java -jar metabase.jar load [my_dump] --mode [skip/update] --on-error [continue/abort]`
 
-The `--mode` flag lets you specify what to do when encountering a duplicate dashboard, question, or any Admin Panel settings that already set (again, except for permissions and user settings, which are not currently included in data dumps). It can either `skip` that item and do nothing to it, or `update` it with the version being loaded. The default is `skip`.
+The `--mode` flag lets you specify what to do when encountering a duplicate dashboard, question, or any Admin Panel settings that already set (again, except for permissions and user settings, which are not currently included in data dumps). It can either `skip` that item and do nothing to it, or `update` it with the version being loaded. The default is `skip`. For example, if you change the order, add or remove any cards in a dashboard that has been loaded previously, mode `skip` won't do any modifications to it, as the load process "skips" any object that previously exists no matter if there are changes on the source file.
 
 The `--on-error` flag allows you to specify whether the load process should keep going or stop when there's an error. The default is `continue`. Note that `abort` won't undo any successful artifact loads that happened before an error was encountered.
 
 Both of these flags are optional.
 
+### Current limitations
+
+- The serialization feature is not able to identify a card in the destination if the visualization settings in the source have changed
```

Comment:
That doesn't sound correct. Might be applicable to dashcards, but not cards. And what does "not able to identify" mean?
If this is one of the new issues, then it's not a limitation, but a bug and should not be listed in the docs.

Reply:
Correct, it's a dashcard. Agree that it's a bug, but as it's a difficult one and might not be fixed in the short term, I wanted to raise some red flag here so users can take this into consideration when using the feature.

Reply:
There's currently 74 other Correctness issues, we don't list those in the docs - and shouldn't. Then our docs would be filled with "limitations".